action.yml

```diff
@@ -54,6 +54,15 @@ runs:
         fi
         echo "MAVEN_OPTS=$MAVEN_OPTS" >> $GITHUB_ENV
 
+    - name: Prepare maven arguments
+      shell: bash
+      run: |
+        if [[ ${{ inputs.build-modules }} != '' ]]; then
+          echo "ADDITIONAL_MAVEN_ARGUMENTS=--also-make" >> $GITHUB_ENV
```

Comment:
instead of **GITHUB_ENV** is better use **GITHUB_OUTPUT** because i don't think this is necessary variable which need be propagated to  GH workflow which use this action